src/arch/mpi-bluegeneq/charmrun

```diff
@@ -3,6 +3,8 @@
 # good for job less than 30 minutes
 # have to pass bank using -A or set PROJECT environment variable
 
+export FROM_CHARMRUN='1'
```

Comment:
What's the purpose of these `FORM_CHARMRUN`, `FROM_CHARMC` variables?

Reply:
See the if in `pathstub.sh`